index.bs

```diff
@@ -65,16 +65,19 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
+The device orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
 
-Orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
-of [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer).
+The device orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
+of the [=low-level=] motion sensors (accelerometer, gyroscope). The type of fusion (complementary
```

Comment:
Game Orientation Vector has nothing really to do with Device orientation vectors. It is not just a power optimization - it works quite differently. 

A Device Orientation Vector is not useful for gaming control for cases like controlling an airplane, because you want it to be relative to the most common stable position and you want that position to adjust over time (ie drift with the gyroscope rotation around z).

The name is bad, because a Device Orientation Vector might still work fine for games, such as the maze game you mentioned which are stationary. But if I am controlling a car or so in a Web VR game where I might turn around and drive in different directions, then that would be useless, as I could only control changes left/right to the northern direction.

The MS name for this is Relative Orientation Sensor, as it is relative to your current stable position (or it adjusts to become).